### PR TITLE
Implement ftu and welcome back flow

### PR DESCRIPTION
**Pull request by Tommytrg**
[code]: https://github.com/witnet/sheikah/blob/master/.github/CODE_OF_CONDUCT.md
[cont]: https://github.com/witnet/sheikah/blob/master/.github/CONTRIBUTING.md
[style]: https://github.com/witnet/sheikah/blob/master/docs/STYLEGUIDE.md

### CODE REVIEW

File: src/components/WelcomeBack/WalletList.vue [hunk 1]
```diff
@@ -0,0 +1,115 @@
+<template>
+    <div class="content">
+      <p class="text">
+        Select which wallet do you want to unlock
+      </p>
+      <ul class="list">
+        <li v-for="wallet in wallets" :key="wallet.id">
+          <router-link :to="`/welcome-back/unlock/${wallet.id}`">
```
Comment (kengork) [inline src/components/WelcomeBack/WalletList.vue]: if we add a router-link, we should also add a css class to remove text-decoration

File: src/views/WelcomeBack.vue [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+      }
+
+      a {
+        text-decoration:none;
+      }
+    }
+  }
+}
+
+// .selection-container {
```
Comment (kengork) [inline src/views/WelcomeBack.vue]: remove unused comments

File: src/components/steps/WalletSeedTypeSelection.vue [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    <div class="selection-container">
+      <h5 class="selection-title">Create your Wallet</h5>
+    </div>
+    <div class="content-wallet">
+    <p class="create-wallet-text">
+      We will now help you create a new local wallet.
+    </p>
+    <ul class="options">
+      <li>
+        <router-link to="/ftu/disclaimer">
```
Comment (kengork) [inline src/components/steps/WalletSeedTypeSelection.vue]: router-link needs to have the CSS class '.router-link'

File: src/components/steps/WalletEncryptionPassword.vue [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  >
+    <div class="content">
+      <p class="paragraph">
+        <strong>PLEASE NOTE:</strong> this password encrypts your Witnet wallet only on this
+        computer. This is not your backup and you cannot restore your wallet with this password.
+        Your seed phrase is still your ultimate backup.
+      </p>
+      <div class="form-row">
+        <label class="label">Password</label>
+        <input class="password-input" type="password" v-model="password">
```
Comment (kengork) [inline src/components/steps/WalletEncryptionPassword.vue]: I think we should use 'element' for the input and add ours in the following issues

File: src/router.js [hunk 5]
```diff
@@ -19,11 +19,30 @@ import WalletSeedValidation from '@/components/steps/WalletSeedValidation.vue'
...
 import Home from '@/views/Home.vue'
+import WelcomeBack from '@/views/WelcomeBack.vue'
+import WalletList from '@/components/WelcomeBack/WalletList.vue'
+import UnlockWallet from '@/components/WelcomeBack/UnlockWallet.vue'
 
 import store from '@/store'
 
 Vue.use(Router)
 
+function checkWalletIsOpen (to, from, next) {
```
[the remark this answers is not in the document]
Comment (Tommytrg) [reply src/router.js]: We need _to_ and _from_ because we need _next_ argument

**Pull request merged**